pkg/client/restclient/versions.go

```diff
@@ -46,10 +46,11 @@ func ServerAPIVersions(c *Config) (groupVersions []string, err error) {
 	configCopy := *c
 	configCopy.GroupVersion = nil
 	configCopy.APIPath = ""
-	baseURL, _, err := defaultServerUrlFor(&configCopy)
+	baseURLs, _, err := defaultServerUrlsFor(&configCopy)
 	if err != nil {
 		return nil, err
 	}
+	baseURL := baseURLs[0]
```

Comment:
@justinsb somehow your comment was lost, this is executed during controller-manager startup. we are usually running api and controller-manager on the same node, configured to use local api server.

I guess i should change this as well, but it is not as critical as kube-proxy/kubelet parts. I hope it is not a problem if i will open issue for this problem and close it with small atomic pull request?